File: scaffold/src/main/java/top/xuqingquan/web/JsAccessEntrace.java
package top.xuqingquan.web;

import com.tencent.smtt.sdk.ValueCallback;
import top.xuqingquan.web.x5.QuickCallJs;

public interface JsAccessEntrace extends QuickCallJs {

    void callJs(String js, ValueCallback<String> callback);

    void callJs(String js);

}

File: scaffold/src/main/java/top/xuqingquan/web/JsAccessEntrace.kt
package top.xuqingquan.web

import com.tencent.smtt.sdk.ValueCallback
import top.xuqingquan.web.x5.QuickCallJs

interface JsAccessEntrace : QuickCallJs {

    fun callJs(js: String, callback: ValueCallback<String>)

    fun callJs(js: String)

}
